.github/workflows/docker-integration-test.yaml

```diff
@@ -1,24 +1,27 @@
 name: docker-integration-test
-on: [push]
+on: [pull_request, push]
 jobs:
   docker-integration-test:
     runs-on: ubuntu-latest
     steps:
       - 
         name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+      -
+        name: Set up Go
+        uses: actions/setup-go@v5
```

Comment:
The go version in the Actions VM is usually pretty old. We used setup-go in the release action. Now we use it in all of them